test/unit/route-reading.js

```diff
@@ -37,14 +37,14 @@ describe('Reading from $route service', function() {
   
   describe('after setting readFromRoute', function() {
     beforeEach(module(function(AnalyticsProvider, $provide){
-      AnalyticsProvider.readFromRoute(true);
+      AnalyticsProvider.readFromRoute(true)
+              // Use classic analytics because have no idea how to track events for the universal one :D
```

Comment:
Because this does work for both classic and universal, it would be great if you could add a complementing set of tests that use universal analytics to ensure the behavior is as expected in both.